app/models/protocol.rb

```diff
@@ -85,6 +85,8 @@ class Protocol < ApplicationRecord
   attr_accessor :bypass_rmid_validation
   attr_accessor :bypass_stq_validation
 
+  mattr_accessor :rmid_server_down
```

Comment:
Semi-caching here to allow us to see if the server is down based on calls made during validations